docs/dataflows.rst

```diff
@@ -0,0 +1,134 @@
+.. _data-flows:
+
+==========
+Data flows
+==========
+
+.. contents::
+   :local:
+
+
+Position data
+=============
+
+Position data is stored in the database in shard tables:
+
+* ``blue_shard_*``
+* ``cell_*``
+* ``wifi_shard_*``
+
+This data is created and updated from incoming API requests.
+
+Data flow:
+
+1. User submits data to one of the submit API endpoints.
+
+   If the user used an api key and the key is sampling submissions, then the
+   submission might get dropped at this point.
+
+   *OR*
+
+   User submits query to one of the locate API endpoints.
+
+   If the query is handled by ``InternalPositionSource``, then the web frontend
+   adds a submission.
+
+2. If the sumission is kept, then the web frontend adds an item to the
+   ``update_incoming`` queue in Redis.
+
+   The item looks like a::
+
+       {"api_key": key, "report": report, "source": source}
+
+   "source" can be one of:
+   
+   * ``gnss``: Global Navigation Satellite System based data
+   * ``fused``: observation data positioned based on fused data
+   * ``fixed``: outside knowledge about the true position of the station
+   * ``query``: position estimate based on query data
+
+3. The Celery scheduler schedules the ``update_incoming`` task every 
+   X seconds--see task definition in `ichnaea/data/tasks.py
```

Comment:
```suggestion
   32 seconds--see task definition in `ichnaea/data/tasks.py
```

Reply:
I didn't want to duplicate the number in two places because it's such an odd number I figured we'd change it. That's why I pointed to the code. I think I want to leave it as X for now.